src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -101,6 +101,9 @@ const ReleaseServiceStatus = ({
         switch (status) {
           case 'NotStarted':
             return { color: 'blue', text: 'Not Started' };
+          case 'Scheduled':
+          case 'Superseded':
```

Comment:
Not sure the front end should ever get superseded as it's immediately been replaced by a more recent status row. Would be interested to know if this happens.

Reply:
I'll turn it into an error.